src/olcs/FeatureConverter.js

```diff
@@ -723,7 +723,8 @@ class FeatureConverter {
         color,
         scale,
         heightReference,
-        position
+        position,
+        ...feature.bbOptions
```

Comment:
You can achieve the same without introducing destructuring with `Object.assign(bbOptions, feature.get('cesiumOptions'));`.

Reply:
Yup, I'll make this update.